backend_jobs/pipeline_utils/database_schema.py

```diff
@@ -0,0 +1,82 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+"""
+
+COLLECTION_IMAGES = 'Images'
+# constant fields names:
+COLLECTION_IMAGES_FIELD_INGESTED_PROVIDERS = 'ingestedProviders'
+COLLECTION_IMAGES_FIELD_INGESTED_RUNS = 'ingestedRuns'
+COLLECTION_IMAGES_FIELD_PROVIDER_ID = 'providerId'
+COLLECTION_IMAGES_FIELD_COORDINATES = 'coordinates'
+COLLECTION_IMAGES_FIELD_DATE_INGESTED = 'dateIngested'
+COLLECTION_IMAGES_FIELD_DATE_SHOT = 'dateShot'
+COLLECTION_IMAGES_FIELD_DATE_FIELDS = 'date'
+COLLECTION_IMAGES_FIELD_IMAGE_ATTRIBUTES = 'imageAttributes'
+COLLECTION_IMAGES_FIELD_FORMAT = 'format'
+COLLECTION_IMAGES_FIELD_RESOLUTION = 'resolution'
+COLLECTION_IMAGES_FIELD_WIDTH = 'width'
+COLLECTION_IMAGES_FIELD_HEIGHT = 'height'
+COLLECTION_IMAGES_FIELD_ATTRIBUTION = 'attribution'
+COLLECTION_IMAGES_FIELD_LABELS = 'labels'
+COLLECTION_IMAGES_FIELD_RANDOM = 'random'
+COLLECTION_IMAGES_FIELD_URL = 'url'
+COLLECTION_IMAGES_FIELD_HASHMAP = 'hashmap'
+COLLECTION_IMAGES_FIELD_VISIBILITY = 'visibility'
+
+COLLECTION_PIPELINE_RUNS = 'PipelineRuns'
+# Constant field names:
+COLLECTION_PIPELINE_RUNS_FIELD_PROVIDER_NAME = 'providerName'
+COLLECTION_PIPELINE_RUNS_FIELD_PROVIDER_ID = 'providerId'
+COLLECTION_PIPELINE_RUNS_FIELD_PROVIDER_VERSION = 'providerVersion'
+COLLECTION_PIPELINE_RUNS_FIELD_PROVIDER_VISIBILITY = 'providerVisibility'
+COLLECTION_PIPELINE_RUNS_FIELD_PIPELINE_RUN_ID = 'pipelineRunId'
+COLLECTION_PIPELINE_RUNS_FIELD_START_DATE = 'startDate'
+COLLECTION_PIPELINE_RUNS_FIELD_END_DATE = 'endDate'
+COLLECTION_PIPELINE_RUNS_FIELD_VISIBILITY = 'visibility'
+
+COLLECTION_IMAGE_PROVIDERS = 'ImageProviders'
+COLLECTION_IMAGE_PROVIDERS_FIELD_API_KEY = 'api key'
```

Comment:
why would every image provider have an api key and secret key?
Maybe something more generic would be to store a map of key/value pairs as a config.